sdk/php/dev/dagger.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "php-sdk-dev",
-  "sdk": "github.com/dagger/dagger/sdk/php",
+  "sdk": "php",
   "source": ".",
-  "engineVersion": "v0.12.3"
+  "engineVersion": "v0.12.7"
```

Comment:
You should update it to `0.13` to benifit from https://github.com/dagger/dagger/pull/8391

Reply:
I do not see it updated, is it done in another PR?